Example/Example/AppleGaugeViewController.swift

```diff
@@ -15,9 +15,9 @@ class AppleGaugeViewController: UIViewController {
     @IBOutlet var gauge_mid: Gauge!
     @IBOutlet var gauge_out: Gauge!
     @IBAction func sliderChanged(_ sender: UISlider) {
-        gauge_in.rate = CGFloat(sender.value * 4)
-        gauge_mid.rate = CGFloat(sender.value * 8)
-        gauge_out.rate = CGFloat(sender.value * 12)
+        gauge_in.rate = CGFloat(sender.value * 10)
+        gauge_mid.rate = CGFloat(sender.value * 10)
+        gauge_out.rate = CGFloat(sender.value * 10)
```

Comment:
I specially set different numbers, to show how to controll speed

Reply:
Let me double check this since a lot of time has passed. I think this fix corresponds to the gauges overdoing the percentage they represent.

Reply:
**This corresponds to values set originally with no changes**. Note that the gauges do not match with the slider progress indication.
![Simulator Screen Shot - iPhone Xʀ - 2019-12-25 at 14 03 01](https://user-images.githubusercontent.com/484008/71448783-83631680-271f-11ea-8cc2-4617ea991aa4.png)

**This corresponds to the changes I made.** Note that the completion of the gauge corresponds to what's displayed on the slider. 
![Simulator Screen Shot - iPhone Xʀ - 2019-12-25 at 14 03 49](https://user-images.githubusercontent.com/484008/71448788-9970d700-271f-11ea-912e-c2402e95f632.png)

Reply:
@skywinder Maybe It's a side-effect kind of bug that you want to look at more in detail.

Reply:
oh, it's not a bug, it's a feature 😄
 I specially made them with different speed, imho animation looks nicer, when they goes with different speed.